packages/ui/resolution/ResolutionDecorator.js

```diff
@@ -116,6 +117,12 @@ const ResolutionDecorator = hoc(defaultConfig, (config, Wrapped) => {
 			this.rootNode = ReactDOM.findDOMNode(this);
 		}
 
+		componentDidUpdate (prevProps) {
+			if (prevProps.fontScale !== this.props.fontScale) {
+				updateBaseFontSize(calculateFontSize(null, this.props.fontScale));
```

Comment:
Is it ok to pass null in calculateFontSize?

Reply:
Yes it is.
If the type value is falsy, the current screenType is used by the getScreenTypeObject function.
In fact, when calling this function in init, type is not passed, so it is passed as undefined.